doc/source/methods/PartialDependence.ipynb

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
<p>Perhaps it would be better to present usage and interpretation first (after a short intro paragraph) and then dive into the definitions and theory? We want people to be able to see quickly what the method can do and the outputs (plots) and roughly how to interpret it. E.g. see</p><p><a href="https://seldon--721.org.readthedocs.build/projects/alibi/en/721/methods/ALE.html" target="_blank">https://seldon--721.org.readthedocs.build/projects/alibi/en/721/methods/ALE.html</a></p><p><a href="https://scikit-learn.org/stable/modules/partial_dependence.html#partial-dependence" target="_blank">https://scikit-learn.org/stable/modules/partial_dependence.html#partial-dependence</a></p><p> </p><p>I would suggest the following (doesn't have to be exact titles/sections):</p><ol><li>Short intro paragraph (like the first one you already have, maybe a bit more, <code>sklearn</code>  page for inspiration)</li><li>Example figure with a high-level description (not as detailed as in the example notebook)</li><li>Usage and API</li><li>Theoretical exposition</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Hope now it looks better</p><p> </p>

Reply:
<p>Looks great, thanks!</p>